server/notification.js

```diff
@@ -129,6 +129,7 @@ class Notification {
             new GoAlert(),
             new ZohoCliq(),
             new GtxMessaging(),
+            new Cellsynt()
```

Comment:
nit: https://github.com/louislam/uptime-kuma/pull/4558#discussion_r1538361780

```suggestion
            new Cellsynt(),
```